Tests/ApolloCacheDependentTests/WatchQueryTests.swift

```diff
@@ -39,7 +39,7 @@ class WatchQueryTests: XCTestCase {
 
       var expectation = self.expectation(description: "Fetching query")
 
-      let watcher = client.watch(query: query) { (result, error) in
+      let watcher = client.watch(query: query, fetchOptions: .POST) { (result, error) in
```

Comment:
I actually think you might get better test results here and in the two bits above if you leave out the `fetchOptions` parameter. That way you can validate that the default option is still `.POST`.

Reply:
You are right, the `watch` was not accepting default value as POST. Just made the changes for it.